### PR TITLE
Added ecs

### PR DESCRIPTION
**Pull request by SirNatan1**
Create an ECS image for Fargate and EC2 + workflow

### CODE REVIEW

File: fluentd/ECS/Json_TaskDefinition [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                }
+            ],
+            "workingDirectory": null,
+            "secrets": null,
+            "dockerSecurityOptions": null,
+            "memory": null,
+            "memoryReservation": 512,
+            "volumesFrom": [],
+            "stopTimeout": null,
+            "image": "docker.io/natan757/ecs_test:1.0.31",
```
Comment (giltk) [inline fluentd/ECS/Json_TaskDefinition:58]: fix image name and version

File: fluentd/ECS/image/fluent.conf [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      key $.docker.name
+      pattern ^(.+)$
+      tag $1.${tag}
+    </rule>
+</match>
+
+<filter **>
+  @type record_transformer
+  <record>
+    tag ${tag}
```
Comment (giltk) [inline fluentd/ECS/image/fluent.conf:63]: remove tag filter

File: fluentd/ECS/readme.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Coralogix Fluentd AWS ECS Image: This folder contains the docker file for the image Coralogix promotes for ECS usecases. fluent.conf - is the default conf, for EC2 usecases. firelens.conf - a configuration ment specificly for Fargate usecases, must be enabled by configuring via Json.
```
Comment (giltk) [inline fluentd/ECS/readme.md:1]: Typo + add firelens context

**Review by giltk — CHANGES_REQUESTED** [review]

File: fluentd/ECS/image/fluent.conf (hunk 2 above)
Comment (SirNatan1) [reply fluentd/ECS/image/fluent.conf]: Deleted

File: fluentd/ECS/readme.md (hunk 3 above)
Comment (SirNatan1) [reply fluentd/ECS/readme.md]: added:
the Json config + added that Firelens will use it's own config if not specified by the Json

File: fluentd/ECS/Json_TaskDefinition (hunk 1 above)
Comment (SirNatan1) [reply fluentd/ECS/Json_TaskDefinition]: changed to: coralogixrepo/fluentd-coralogix-ecs:latest

**Review by giltk — APPROVED** [review]

**Review by giltk — APPROVED** [review]

**Review by talknopf — APPROVED** [review]
approving per gils inspection

**Pull request merged**